package.json

```diff
@@ -10,51 +10,24 @@
   "repository": "your-repository-name",
   "scripts": {
     "start": "probot run ./index.js",
-    "test": "jest && standard && npm run doc-lint"
+    "test": "jest && standard"
```

Comment:
`standard` is failing for me due to trailing spaces `/tests/index.js`. I fixed that in #4